propagator/ottrace/README.md

```diff
@@ -7,8 +7,8 @@ The `opentelemetry-propagator-ottrace` gem contains injectors and extractors for
 
 | Header Name         | Description                                                                                                                            | Required              |
 | ------------------- | -------------------------------------------------------------------------------------------------------------------------------------- | --------------------- |
-| `ot-tracer-traceid` | uint64 encoded as a string of 16 hex characters                                                                                        | yes                   |
-| `ot-tracer-spanid`  | uint64 encoded as a string of 16 hex characters                                                                                        | yes                   |
+| `ot-tracer-traceid` | 64-bit; 16 hex digits or 128-bit; 32 hex digits                                                          | yes                   |
```

Comment:
```suggestion
| `ot-tracer-traceid` | 64-bit; 16 hex digits                                              | yes                   |
```

Reply:
`ot-tracer-traceid` will always be 64-bits on the wire. double check, but i believe this test case confirms that: https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/propagator/ottrace/test/opentelemetry/propagator/ottrace/text_map_propagator_test.rb#L311-L320